examples/flower_classifier/MLproject

```diff
@@ -0,0 +1,21 @@
+name: FlowerClassifier
+
+conda_env: conda.yaml
+
+entry_points:
+  # train Keras DL model
+  train:
```

Comment:
Why isn't this called main? People have to do `mlflow run . -e train` to run this instead of just `mlflow run .`.